tests/testthat/test-chartcolors.R

```diff
@@ -38,6 +38,15 @@ test_that("Alpha color values",
         "#FF000033", check.attributes = FALSE)
 })
 
+test_that("GetVectorOfColors always interpolates when values provided",
```

Comment:
According to the documentation, the colors in `palette.custom.palette` will be recycled, so perhaps that needs to be updated.

The test description refers to interpolation, but it is not clear how the result of `GetVectorOfColors` in the test includes interpolation (as opposed to recycling?)